apps/publisher/controllers/apis-router.jag

```diff
@@ -35,44 +35,47 @@ var mapper = function(path) {
         return path;
     };
 };
-metrics.start("apis-router","router");
-if (uriMatcher.match(API_URL)) {
-    var args = uriMatcher.elements();
-    var tenantId = DEFAULT_TENANT; //Assume that the tenant will be the super tenant
-    //Determine the tenant by the user
-    if (user) {
-        tenantId = user.tenantId;
-    }
-    //Obtain the set of available pages
-    var app = require('rxt').app;
-    var page = getPage(args.suffix);
-    var path = app.getApiEndpointPath(tenantId, page);
-    if (path) {
+try{
```

Comment:
Thanks @splinter , I have fixed it in [this commit](https://github.com/tmkasun/carbon-store/commit/a5917ea67af4d60979a9f5f59636c7672843645b#diff-cd2251c905a3e28c7b8dcb12d1d03602)